src/MetaModels/DcGeneral/Events/Table/Attribute/Subscriber.php

```diff
@@ -519,24 +515,4 @@ public function handleUpdateAttribute(PostPersistModelEvent $event)
             }
         }
     }
-
-    /**
-     * Handle the deletion of an attribute and all attached data.
-     *
-     * @param PreDeleteModelEvent $event The event.
-     *
-     * @return void
-     */
-    public function handleDeleteAttribute(PreDeleteModelEvent $event)
-    {
-        if (($event->getEnvironment()->getDataDefinition()->getName() !== 'tl_metamodel_attribute')) {
-            return;
-        }
-
-        $attribute = $this->createAttributeInstance($event->getModel()->getPropertiesAsArray());
-
-        if ($attribute) {
-            $attribute->destroyAUX();
```

Comment:
You are aware that by removing this line, no database column will be removed anymore?
This will make it impossible to delete attributes and create new ones with the same name again, as the column persists.
Can you explain why you are removing this handler?